marconi-mamba/src/Marconi/Api/HttpServer.hs

```diff
@@ -72,7 +72,7 @@ getTime = timeString <$> liftIO getCurrentTime
 
 getTargetAddresses
     :: DBQueryEnv               -- ^ database configuration
-    ->  Handler (Set Text)
```

Comment:
The sql is already using the txid as unique index. so for all the data that comes from storage, I change to List to save some processing, and change the TargetAddress as well to avoid converting to List.